crates/xtask/src/main.rs

```diff
@@ -494,6 +501,7 @@ impl RunnerOptions {
         let mut cargo = Command::new("cargo");
         let mut rustflags = Vec::new();
         let mut features = self.features.clone();
+        let mut runner_args = Vec::new();
```

Comment:
fyi: Interesting choice. I was thinking only doing this for the host runner, because that's the only one to use `cargo run`, but this is more general in case one day there's another runner that uses `cargo run`.

Reply:
Yeah, I guess I just wanted this new args to be consistent with others that are defined here.

Reply:
Yes, but those args are common to all runners, while passing arguments to `cargo run` is specific to the host runner. And actually I found an issue. If you just want to build the host runner instead of running it you get an error, because `cargo build` doesn't take extra arguments.

```
% cargo xtask runner host --web --web-port=42
cd "crates/runner-host" && "cargo" "build" "--release" "--target=x86_64-unknown-linux-gnu" "--features=debug,web,wasm" "--" "--web-port=42"
```

So I would actually move all the logic inside the `if run && self.name == "host" {` at the end. It's fine to always add the `--` even if nothing follows.

Another change that would be worth it, would be to change `if self.name == "host" && self.web {` to also check if `web_host` or `web_port` is defined, such that setting one of those flags implicitly sets `web`.